haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,10 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+srcGitRev="1da1d52429db53f15c3debf817e982d9ce6346b4"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
there should be a sha256sum command, also trying to build with haikuporter will fail but tell the calculated checksum.